ionoscloud/resource_nfs_cluster_test.go

```diff
@@ -0,0 +1,219 @@
+//go:build all || nfs || nfs_cluster
+
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils/constant"
+)
+
+func TestAccNFSCluster_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"random": {
+				VersionConstraint: "3.4.3",
+				Source:            "hashicorp/random",
+			},
+			"time": {
+				Source:            "hashicorp/time",
+				VersionConstraint: "0.11.1",
+			},
+		},
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckNFSClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckNFSClusterConfig_basic,
```

Comment:
@cristiGuranIonos 
```suggestion
				Config: testAccCheckNFSClusterConfigBasic,
```

Reply:
As a note, we have this style in multiple locations in the code. What is after the `_` is the test step.